File: ps/codeforces/705A.py
def a(n):
    if n ==1:
        return ""
    b = "I love that " if n%2 else "I hate that "
    return a(n-1) + b
n = int(input()) 
b = "I hate it" if bool(n%2) else "I love it"
print(a(n) + b)

File: ps/codeforces/500to1000/705A.cpp
#include <iostream>
using namespace std;

int main() {
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    int n;
    cin >> n;
    cout << "I hate";
    for (int i = 0; i < n - 1; i++) {
        if (i%2) {
            cout << " that I hate";
        } else {
            cout << " that I love";
        }
    }
    cout << " it";

    return 0;
}
